test/reflect.js

```diff
@@ -0,0 +1,483 @@
+
+      // No getter for setter property
+      expect(Reflect.get(object, 'setter', other)).to.equal(undefined);
+    });
+
+    it('will search the prototype chain', function () {
+      var other = Object.create(object);
+      other._value = 17;
+
+      var yet_another = { _value: 4711 };
+
+      expect(Reflect.get(other, 'value', yet_another)).to.equal(4711);
+
+      expect(Reflect.get(other, 'bool', yet_another)).to.equal(true);
+    });
+  });
+
+  describe('Reflect.getOwnPropertyDescriptor()', function () {
+    it('is a function', function () {
+      expect(typeof Reflect.getOwnPropertyDescriptor).to.equal('function');
+    });
+
+    it('throws if the target isn\'t an object', function () {
+      testPrimitiveThrow(function (item) {
+        return Reflect.getOwnPropertyDescriptor(item, 'prop');
+      });
+    });
+  });
+
+  describe('Reflect.getPrototypeOf()', function () {
+    it('is a function', function () {
+      expect(typeof Reflect.getPrototypeOf).to.equal('function');
+    });
+
+    it('throws if the target isn\'t an object', function () {
+      testPrimitiveThrow(function (item) {
+        return Reflect.getPrototypeOf(item);
+      });
+    });
+  });
+
+  describe('Reflect.has()', function () {
+    it('is a function', function () {
+      expect(typeof Reflect.has).to.equal('function');
+    });
+
+    it('throws if the target isn\'t an object', function () {
+      testPrimitiveThrow(function (item) {
+        return Reflect.has(item, 'prop');
+      });
+    });
+
+    it('will detect own properties', function () {
+      var target = Object.create(null);
+
+      expect(Reflect.has(target, 'prop')).to.equal(false);
+
+      target.prop = undefined;
+      expect(Reflect.has(target, 'prop')).to.equal(true);
+
+      delete target.prop;
+      expect(Reflect.has(target, 'prop')).to.equal(false);
+
+      Object.defineProperty(target, 'accessor', {
+        set: function () {}
+      });
+
+      expect(Reflect.has(target, 'accessor')).to.equal(true);
+
+      expect(Reflect.has(Reflect.has, 'length')).to.equal(true);
+    });
+
+    it('will search the prototype chain', function () {
+      var intermediate = Object.create(object),
+      target = Object.create(intermediate);
+
+      intermediate.some_property = undefined;
+
+      expect(Reflect.has(target, 'bool')).to.equal(true);
+      expect(Reflect.has(target, 'some_property')).to.equal(true);
+      expect(Reflect.has(target, 'undefined_property')).to.equal(false);
+    });
+  });
+
+  describe('Reflect.isExtensible()', function () {
+    it('is a function', function () {
+      expect(typeof Reflect.isExtensible).to.equal('function');
+    });
+
+    it('returns true for plain objects', function () {
+      expect(Reflect.isExtensible({})).to.equal(true);
+      expect(Reflect.isExtensible(Object.preventExtensions({}))).to.equal(false);
+    });
+
+    it('throws if the target isn\'t an object', function () {
```

Comment:
feel free to use `’` here to both avoid the need to escape, and to please the typographically inclined